services/elasticache/pkg/resource/cache_parameter_group/custom_api.go

```diff
@@ -52,9 +53,58 @@ func (rm *resourceManager) customSetOutputDescribeCacheParameters(
 		return err
 	}
 	ko.Status.Parameters = parameters
+	rm.customSetOutputSupplementAPIs(ctx, cacheParameterGroupName, ko)
+
 	return nil
 }
 
+func (rm *resourceManager) customSetOutputSupplementAPIs(
+	ctx context.Context,
+	cacheParameterGroupName *string,
+	ko *svcapitypes.CacheParameterGroup,
+) (bool, error) {
```

Comment:
Is `bool` return value here supposed to indicate if something about the `Events` collection *changed*? Or is it supposed to indicate that Events were returned and set on the Status?

Reply:
Removed `bool` return as it wasn't required.